### PR TITLE
Adding ProtocolVersion support to TestHost

### PR DESCRIPTION
**Pull request by rynowak**
/cc @troydai @BillHiebert @vijayrkn @davidfowl

### CODE REVIEW

File: src/Microsoft.Framework.TestHost/Program.cs [hunk 1]
```diff
@@ -22,7 +23,9 @@ public Program(IServiceProvider services)
 
         public int Main(string[] args)
```
Comment (rynowak) [inline src/Microsoft.Framework.TestHost/Program.cs]: Right now there's a bunch of vestigial stuff in this class to support older VS builds. Once VS goes RTM, I'll remove the old support.

**Comment by troydai** [comment]
I'll reviewing.

**Comment by troydai** [comment]
As far as my concern, it looks good. :shipit:

**Comment by troydai** [comment]
@rynowak is this PR merged?

**Comment by rynowak** [comment]
Yes.

**Pull request closed without merging**